wolfssl-3.9.6/README.md

```diff
@@ -0,0 +1,139 @@
+Finds a SEGV and three heap overflow errors in `src/ssl.c`
+in [wolfSSL v.3.9.6](https://www.wolfssl.com/wolfSSL/Home.html).
```

Comment:
> Unfortunately some of the bugs are shallower 

This makes it a bad benchmark. 
I wonder if there is a revision where no shallow bug exists, but deeper bugs are still there. 
Can you find such?